09-minimal-dv-job.md

```diff
@@ -0,0 +1,155 @@
+on.
+
+There are [many configuration 
+attributes](http://lhcb-release-area.web.cern.ch/LHCb-release-area/DOC/davinci/releases/v36r6/doxygen/py/dc/d2f/class_da_vinci_1_1_configuration_1_1_da_vinci.html#ac788f6a80f5f61d47056debe7b86ca71) 
+defined on the `DaVinci` object, but we will only set the ones that are 
+necessary for us.
+
+```python
+from Configurables import DaVinci
+
+# Configure DaVinci
+DaVinci().UserAlgorithms += [dtt]
+DaVinci().InputType = 'DST'
+DaVinci().TupleFile = 'DVntuple.root'
+DaVinci().PrintFreq = 1000
+DaVinci().DataType = '2011'
+DaVinci().Simulation = False
+# Only ask for luminosity information when not using simulated data
+DaVinci().Lumi = not DaVinci().Simulation
+DaVinci().EvtMax = -1
+```
+
+Nicely, a lot of the attributes of the `DaVinci` object are self-explanatory: 
+`InputType` should be `'DST'` when giving DaVinci DST files; `PrintFreq` 
+defines how often DaVinci should print its status; `DataType` is the year of 
+data-taking the data corresponds to; `Simulation` should be `True` when using 
+Monte Carlo data, `Lumi` defines whether to store information on the integrated 
+luminosity the input data corresponds to; and `EvtMax` defines how many events 
+to run over, where a value of `-1` means "all events".
+
+In order to run an algorithm that we have previously created, we need to add it 
+to the `UserAlgorithms` list.
+The `TupleFile` attribute defines the name of the ROOT output file that DaVinci 
+will store any algorithm output in, which should be our ntuple.
+
+All that's left to do is to say what data we would like to run over.
+As we already have a data file [downloaded locally](03-bookkeeping.html), we 
+define that as our input data.
+
+```python
+from GaudiConf import IOHelper
+
+# Use the local input data
+IOHelper().inputFiles([
+  './00041838_00047508_1.charmcompleteevent.dst'
+], clear=True)
+```
+
+This says to use the `.dst` file that is in the same directory as the options 
+file, and to clear any previous input files that might have been defined.
+
+That's it! We're ready to run DaVinci.
+
+In the same folder as your options file `ntuple_options.py` and your DST file 
+ending in `.dst`, there's just a single command you need run on `lxplus`.
+
+```shell
+$ lb-run DaVinci v36r6 gaudirun.py ntuple_options.py
+```
+
+The full options file we've created, `ntuple_options.py`, is [available 
+here](), and the resulting ntuple is [available here]().
+
+> ## Using a microDST {.callout}
+>
+> A microDST (or µDST) is a smaller version of a DST.
+> Some stripping lines go to µDSTs, and some go to DSTs.
+> There are two things that need changing it our options file in order to have 
+> in work when it is used with a stripping line that goes to a µDST:
+>
+> 1. The `DecayTreeTuple.Inputs` attribute should start at the word
+>    `Phys`; and
+> 2. The `RootInTES` attribute on the `DaVinci` object to be set to 
+>    `/Event/STREAM`
```

Comment:
Make this $STREAM or some such to indicate it is a variable, not the literal word STREAM